javascripts/discourse/components/top-contributors.js

```diff
@@ -4,17 +4,23 @@ import { tracked } from "@glimmer/tracking";
 
 export default class TopContributors extends Component {
   @tracked topContributors = null;
+  order = null;
+  viewAllUrl = null;
 
   constructor() {
     super(...arguments);
-
     const count = this.args?.params?.count || 5;
+    this.order = this.args?.params?.order || "likes_received";
```

Comment:
This can be initialized outside the constructor, e.g.

```js
export default class TopContributors extends Component {
  order = this.args.params?.order || "likes_received";